crates/project/src/project.rs

```diff
@@ -3160,8 +3162,10 @@ impl Project {
         language_server
             .on_request::<lsp::request::ShowMessageRequest, _, _>({
                 let this = this.clone();
+                let name = name.to_string();
                 move |params, mut cx| {
                     let this = this.clone();
+                    let name = name.to_string();
```

Comment:
I'd rather not do this double-capture-clone dance, but I was unable to use the server ID to grab the LSP from the project.